src/unit_tests/os_crypto/shared/test_keys.c

```diff
@@ -15,9 +15,6 @@
 #include <stdlib.h>
 
 #include "../headers/shared.h"
-#include "../headers/sec.h"
-#include "../../wrappers/common.h"
-#include "../../wrappers/wazuh/shared/debug_op_wrappers.h"
```

Comment:
Is this modification necessary?

Reply:
Cleaning unnecessary headers improves compilation times.

Reply:
Ditto

Reply:
Done